components/layout/ModPage.vue

```diff
@@ -490,8 +489,11 @@ export default {
   }
   .info {
     @extend %column;
+    padding: 1rem;
+    display: flex;
+    justify-content: flex-start;
```

Comment:
![image](https://i.imgur.com/FJYskNd.png)

Reply:
It also looks a bit strange on mobile, with the opposite problem:
![image](https://i.imgur.com/SO4wCuf.png)
The `Mods > Mod Name > Description` should be centred between that rather large gap and that sort of lack of gap